contracts/leverage_trading/src/view.rs

```diff
@@ -265,6 +265,97 @@ impl Contract {
     pub fn view_max_position_amount(&self) -> U128 {
         U128(self.max_order_amount)
     }
+
+    /// this method uses the mock data
+    pub fn get_total_pending_orders_per_pair(&self, _pair_id: PairId) -> U128 {
+        U128(3)
+    }
+
+    /// his method uses the mock data
+    pub fn get_pending_orders(
+        &self,
+        _pair_id: PairId,
```

Comment:
@vlad-sundunchan it's better to keep those parameters with normal names (without leading underscore) because it's an interface of the method, and once you change parameters names all users of this interface have to change it too

check `#[allow(unused_variables)]` or similar `suppress` instructions